docs/adr/adr_monorepo.md

```diff
@@ -13,7 +13,7 @@ proposed
 ## Context or Problem Statement
 
 We are an all remote team with only bi-weekly or monthly team calls. Therefore, communication between the dev team members and transparency for all members is of the highest priority. At the moment, we have the frontend and backend code separated in two repos. We already see that the documentation is not in the same state on both repos and not all the information is in one place. The instructions to set-up the dev environment is distributed and is confusing for new people.
-For a frontend and backend devs coder can work on small tasks completly separate from each other. However, bigger tasks are tougher to coordinate. 
+In a monorepo architecture, frontend and backend developers can work on small tasks completly separately from each other, and coordinate more easily for more complex, far-reaching task.  
```

Comment:
should say "FE and BE developers can **still** work on small tasks completely separate from each other"